cmd/thermal-recorder/config.go

```diff
@@ -88,7 +98,12 @@ type rawConfig struct {
 	Turret       TurretConfig `yaml:"turret"`
 }
 
+var defaultUploaderConfig = rawUploaderConfig{
+	DeviceName: "NotSet",
```

Comment:
Maybe leave it as empty, and then don't add the field to the CPTV file if it's not available?